gocourse02/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	zoo "gocourse02/zoo"
+)
+
+func main() {
+	lion := zoo.NewLion(zoo.NewAnimal("Mykola", zoo.Male), "green")
+	lioness := zoo.NewLion(zoo.NewAnimal("Natalia", zoo.Female), "white")
+	cub := lion.Reproduce(*lioness)
+	cub.Show()
+
+	zooKeeper := zoo.NewZookeeper("Mykyta", []zoo.Reproducer{})
```

Comment:
Старатись писати консистентно. Або скрізь `zooKeeper` (два слова), або скрізь `zookeeper` (одне).

```suggestion
	zookeeper := zoo.NewZookeeper("Mykyta", []zoo.Reproducer{})
```